BlueprintUI/Sources/Blueprint View/BlueprintView.swift

```diff
@@ -419,3 +432,65 @@ extension BlueprintView {
     }
 }
 
+extension BlueprintView {
+    private func logLayoutStart() {
+        if #available(iOS 12.0, *) {
+            os_signpost(
```

Comment:
Fwiw in the past (and when I took a stab at signpost logging in a draft PR), I wrapped up the logging into a little helper so all the os_signpost stuff was in one place: https://github.com/square/Blueprint/pull/174/files#diff-34f7283a6db50334cae24989cc3fd7656859bae36d059ba5bb7a35206d3017e4R2

Reply:
> Fwiw in the past (and when I took a stab at signpost logging in a draft PR), I wrapped up the logging into a little helper so all the os_signpost stuff was in one place: https://github.com/square/Blueprint/pull/174/files#diff-34f7283a6db50334cae24989cc3fd7656859bae36d059ba5bb7a35206d3017e4R2

Sure, I can centralize these.